fastapi/openapi/models.py

```diff
@@ -5,13 +5,16 @@
 from pydantic import BaseModel, Schema as PSchema
 from pydantic.types import UrlStr
 
+logger = logging.getLogger("fastapi")
+logger.addHandler(logging.NullHandler())
```

Comment:
Please remove this line. If we add the `NullHandler` then, by default, the warning is never shown in the output.

The docs say:

> It is strongly advised that you do not add any handlers other than `NullHandler` to your library’s loggers.

But that doesn't mean that they advice to add the `NullHandler`, just that if we wanted to add something, it should only be the `NullHandler`.

In this case, we can just not add any handler to keep it showing in output, warning developers that email fields won't be validated.

Reply:
Also, if they want to remove the warning, they can get the `fastapi` logger, and add the `NullHandler` themselves, thanks to this PR.

Reply:
fixed, you're right, as usual :)

Reply:
Thanks! :grin: